id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/database/IdentifiersDaoTest.scala

```diff
@@ -31,17 +31,32 @@ class IdentifiersDaoTest
     }
 
     it("should return a future of None if looking up a non-existent Miro ID") {
-      whenReady(identifiersDao.lookupMiroID("A missing mouse")) { maybeIdentifier =>
-        maybeIdentifier shouldNot be(defined)
-      }
+      assertLookupMiroIDFails(
```

Comment:
I find the name of this function a little bit confusing cause it's not actually failing... Maybe `assertLookupMiroIDReturnsNone`?